recipe/meta.yaml

```diff
@@ -23,18 +22,33 @@ requirements:
   run:
     - python >={{ python_min }},<3.13
     - apache-airflow >=3.0.2
-    - apache-airflow-providers-common-compat >=1.2.1
+    - apache-airflow-providers-common-compat >=1.8.0
+    # more restrictive for python <3.13
     - blinker >=1.6.2
+    # more restrictive for python <3.13
     - flask >=2.2.1,<2.3
-    - flask-appbuilder ==4.6.3
+    - flask-appbuilder ==5.0.1
+    # more restrictive for python <3.13
     - flask-login >=0.6.2
-    - flask-session >=0.4.0,<0.6
+    # more restrictive for python <3.13
+    - flask-session >=0.8.0
+    # more restrictive for python <3.13
+    - msgpack >=1.0.0
+    # more restrictive for python <3.13
+    - flask-sqlalchemy >=3.0.5
+    # more restrictive for python <3.13
+    - sqlalchemy >=1.4.36,<2
```

Comment:
Same inability to simultaneously satisfy the flask-sqlalchemy and sqlalchemy requirements as in #33 and #34.